pep-0675.rst

```diff
@@ -534,10 +527,6 @@ match:
 Backwards Compatibility
 =======================
 
-``Literal[str]`` is acceptable at runtime, so
```

Comment:
Could instead add a note about adding `LiteralString` to `typing_extensions`.

Reply:
Ok, I've added a line. Lmk if that looks fine.

Reply:
Thanks, looks good